source/beamline/BioXAS/BioXASSideBeamStatusControl.cpp

```diff
@@ -0,0 +1,185 @@
+#include "BioXASSideBeamStatusControl.h"
+#include "actions3/AMActionSupport.h"
+
+BioXASSideBeamStatusControl::BioXASSideBeamStatusControl(const QString &name, QObject *parent) :
+	BioXASBeamStatusControl(name, parent)
+{
+	// Initialize inherited variables.
+
+	setContextKnownDescription("SideBeamStatus");
+
+	// Initialize local variables.
+
+	frontEndStatus_ = 0;
```

Comment:
I noticed that you have three statuses here but in methods like `shouldStop()` you only ever use the `soeBeamStatus_`.  Is there a particular reason for only using the one status?